### DISCUSSION TITLE
Paypal and Github sponsors

### DISCUSSION THREAD

**Discussion by mo8it** [JuliaLang/julia#48752] [General]
I sponsor Julia on Github, but since Paypal is removed as a payment method starting with tomorrow, I don't have another option for donation because I don't have a credit card. Could you please add an alternative for people (especially in EU) that want to support with some money?

I would personally recommend Liberapay: https://liberapay.com
It is free and open source and has renewing donations :)

**Comment by DilumAluthge** [comment]
If you donate to Julia via NumFOCUS, you can use PayPal as a payment method: https://numfocus.org/donate-to-julia
